src/common/pubsub.ts

```diff
@@ -332,3 +334,10 @@ export const notifyUserCreated = async (
 
 export const notifyFeaturesReset = async (log: EventLogger): Promise<void> =>
   publishEvent(log, featuresResetTopic, {});
+
+export type ContentRequested = Pick<ArticlePost, 'id' | 'url' | 'origin'>;
+
+export const notifyContentRequested = async (
+  log: EventLogger,
+  content: ChangeObject<ContentRequested>,
```

Comment:
We don't need the `ChangeObject` in this case, right?

Reply:
Ah not particular indeed, we can make it a set ContentRequested type will address this.